ntoskrnl/lpc/port.c

```diff
@@ -277,7 +276,7 @@ NTAPI
 NtQueryPortInformationProcess(VOID)
 {
     /* This is all this function does */
-    return STATUS_UNSUCCESSFUL;
+    return TRUE;
```

Comment:
If this function really returns a boolean value then you must change the function's declaration to a `BOOLEAN` as well.

Reply:
Yes, you are right. And this should be fixed in NDK as well.

Reply:
All documentation (including public one like MSDN) mentions that this function's return type is NTSTATUS.

Reply:
Yes, I have found out that this function returns "1", thus TRUE seemed the valid return value here. I have not verified the function declaration to be honest, but changed the type as suggested by @Fraizeraust. This seems to be my mistake, as I googled for that now and indeed seems that it should be NTSTATUS...